### PR TITLE
Decode HEX Encoded UTXO

### PR DESCRIPTION
**Pull request by null-ref-ex**
- why not make this easy given it's the default response format from the RPC API now

### CODE REVIEW

File: src/utils/errors.ts [hunk 1]
```diff
@@ -43,6 +43,7 @@ const SUBNET_ID_ERROR_CODE: string = "1041"
 const TYPE_NAME_ERROR_CODE: string = "1042"
 const SUBNET_THRESHOLD_ERROR_CODE: string = "1043"
 const SUBNET_ADDRESS_ERROR_CODE: string = "1044"
+const UNKNOWN_FORMAR_ERROR_CODE: string = "1045"
```
Comment (cgcardona) [inline src/utils/errors.ts:46]: Fix typo. `UNKNOWN_FORMAR_ERROR_CODE` should be `UNKNOWN_FORMAT_ERROR_CODE`

File: src/utils/errors.ts [hunk 2]
```diff
@@ -372,8 +373,15 @@ export class SubnetAddressError extends AvalancheError {
   }
 }
 
+export class UnknownFormatError extends AvalancheError {
+  constructor(m: string) {
+    super(m, UNKNOWN_FORMAR_ERROR_CODE)
```
Comment (cgcardona) [inline src/utils/errors.ts:378]: Fix typo. `UNKNOWN_FORMAR_ERROR_CODE` should be `UNKNOWN_FORMAT_ERROR_CODE`

File: src/apis/platformvm/utxos.ts [hunk 3]
```diff
@@ -88,18 +89,31 @@ export class UTXO extends StandardUTXO {
...
-  fromString(serialized: string): number {
-    /* istanbul ignore next */
-    return this.fromBuffer(bintools.cb58Decode(serialized))
+  fromString(serialized: string, format: string = 'cb58'): number {
+    switch (format) {
+      case "cb58": {
+         /* istanbul ignore next */
+        return this.fromBuffer(bintools.cb58Decode(serialized));
+      };
+      case "hex": {
```
Comment (cgcardona) [inline src/apis/platformvm/utxos.ts:108]: Please add test coverage for `"hex"` case.

**Review by cgcardona — DISMISSED** [review]
Changes requested:

1. Add test coverage
2. Fix typos
3. Merge PR into `development` branch instead of `master`

Thanks!

File: src/utils/errors.ts (hunk 2 above)
Comment (null-ref-ex) [reply src/utils/errors.ts:378]: Good spot, fixed.

File: src/apis/platformvm/utxos.ts (hunk 3 above)
Comment (null-ref-ex) [reply src/apis/platformvm/utxos.ts:108]: Added, if not enough please shout and I can add more.

**Review by dfenstermaker — APPROVED** [review]

**Pull request merged**